src/components/manage/Sidebar/AlignBlock.jsx

```diff
@@ -20,19 +20,26 @@ const messages = defineMessages({
     id: 'Center',
     defaultMessage: 'Center',
   },
+  wide: {
+    id: 'Full',
+    defaultMessage: 'Full',
+  },
```

Comment:
These need a different id and defaultMessage, it is currently equal to `full`, the one right below

Reply:
Copy paste issues :)